routes/users.js

```diff
@@ -536,6 +536,19 @@ router.delete('/user/:id', async function(req, res) {
             res.status(403).send({ message: 'User owns some web sites, please change owner first!' });
             return;
         }
+        let projects = await dbsrv.mongo_projects().find({ owner: uid }).toArray();
+        if (projects && projects.length > 0) {
+            res.status(403).send({ message: 'User owns some projects, please change owner first!' });
+            return;
+        }
+        const allprojects = user.projects ? user.projects : [];
+        for (let project_name of allprojects) {
+            const users_in_project = await dbsrv.mongo_users().find({ 'projects': project_name }).toArray();
+            if (users_in_project.length <= 1) {
+                res.status(403).send({ message: 'User is the last member of project ${project_name}' });
```

Comment:
Might be better to store project names somewhere, and send the response after will all projects.
(Otherwise, you'll try to delete -> change a project -> try to delete -> etc)